korman/properties/modifiers/water.py

```diff
@@ -464,7 +464,7 @@ class PlasmaWaveTexState(PlasmaWaveState, PlasmaModifierProperties):
 
     _min_length_default = 0.1
     _max_length_default = 4.0
-    _amplitude_default = 10
+    _amplitude_default = 0.040
```

Comment:
`0.1`?

Reply:
This setting was around average of what I've seen in most of the wavesets. This default creates a rather nice ripple effect (such as the ocean in Noloben) that Age builders can then play around with from there.

Reply:
Looking at the old AlcScript for wavesets (yeah, I went a bit cross-eyed looking at *that* stuff again 😜 ), an argument could be made for setting the default at 0.010.